source/MRMesh/MRVoxelGraphCut.cpp

```diff
@@ -149,8 +149,18 @@ void VoxelGraphCut::buildInitialForest_( const VoxelBitSet & sourceSeeds, const
         minPathCapacity.setLargerValue( v, FLT_MAX );
     }
 
-    for (;;)
+    float progress = 0.0f;
+    const float targetProgress = 0.5f;
+
+    for ( int i = 0; ; ++i )
     {
+        if ( cb )
```

Comment:
pls return some `i % 100 == 0` in this condition, it was confusing with `cb( 0.0f )` but now it will be ok